exercises/practice/least_change/least_change_test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from z3 import *
+from .least_change import *
+
+# Tests adapted from https://github.com/exercism/python/blob/main/exercises/practice/change/change_test.py
+
+class ChangeTest(unittest.TestCase):
+    def test_single_coin_change(self):
+        self.assertEqual(str(least_change([1, 5, 10, 25, 100], 25)), "[10 = 0, 25 = 1, 100 = 0, 5 = 0, min_coins = 1, "
```

Comment:
Is there a reason you're converting it to a string rather than making a direct comparison between two lists? Seems a little error prone if the formatting was to change